src/__tests_/App.test.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
you can add 
```
import React from 'react';
global.React = React; 
```
to `jest.config.js` to set this for all test files so we don't have to import it everytime.